tests/cmdline/commands/test_node_list_relationship.py

```
@@ -0,0 +1,73 @@
+    node_b.store()
+    
+    node_c = orm.Data()
+    node_c.label = 'node_c'
+    node_c.base.links.add_incoming(node_b, link_type=LinkType.CREATE, link_label='link_bc')
+    node_c.store()
+    
+    return node_a, node_b, node_c
+
+def test_node_list_ancestor(run_cli_command, provenance_graph):
+    """Test the `--ancestor` filter."""
+    node_a, node_b, node_c = provenance_graph
```

Ruff (0.15.21)

[warning] Unpacked variable `node_b` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)